PIL/TiffImagePlugin.py

```diff
@@ -225,6 +225,75 @@ def _limit_rational(val, max_val):
 _load_dispatch = {}
 _write_dispatch = {}
 
+class IFDRational(Fraction):
+    """ Implements a rational class where 0/0 is a legal value to match
+    the in the wild use of exif rationals.
+
+    e.g., DigitalZoomRatio - 0.00/0.00  indicates that no digital zoom was used
+    """
+
+    """ If the denominator is 0, store this as a float('nan'), otherwise store
+    as a fractions.Fraction(). Delegate as appropriate
+
+    """
+    
+    __slots__ = ('numerator', 'denominator', '_val') 
+
+    def __init__(self, value, denominator=1):
+        """
+        :param value: either an integer numerator, a
+        float/rational/other number, or an IFDRational
+        :param denominator: Optional integer denominator
+        """
+        self.denominator = denominator
+        self.numerator = value
+        
+        if type(value) == IFDRational:
+            self.denominator = value.denominator
+            self.numerator = value.numerator
+            self._val = value._val
+            return
+
+        if denominator == 0:
+            self._val = float('nan')
+            return
+
+        try:
+            if denominator == 1:
+                self._val = Fraction(value)
+            else:
+                self._val = Fraction(value, denominator)
```

Comment:
This is the test failure from python 2.6
 ```
======================================================================
ERROR: TestFileTiff.test__limit_rational_float
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/python-pillow/Pillow/Tests/test_file_tiff.py", line 310, in test__limit_rational_float
    ret = _limit_rational(value, 65536)
  File "/home/travis/build/python-pillow/Pillow/PIL/TiffImagePlugin.py", line 219, in _limit_rational
    n_d = IFDRational(1 / val if inv else val).limit_rational(max_val)
  File "/opt/python/2.6.9/lib/python2.6/fractions.py", line 99, in __new__
    numerator = operator.index(numerator)
TypeError: 'float' object cannot be interpreted as an index
```
so the numerator in this situation `value` should be cast into an int please